surface_temperatures/create_artifact.jl

```diff
@@ -0,0 +1,104 @@
+using Downloads
+using NCDatasets
+using ClimaArtifactsHelper
+
+const FILE_URL = "https://berkeley-earth-temperature.s3.us-west-1.amazonaws.com/Global/Gridded/Land_and_Ocean_LatLong1.nc"
+
+const FILE_NAME = "Land_and_Ocean_LatLong1.nc"
+
+function download_progress(total::Integer, now::Integer)
+    if total == 0
+        print("Downloaded $(round(now/10^9, digits=2)) GB \r")
+    else
+        print(
+            "Downloaded $(round(now/10^9, digits=2)) out of $(round(total/10^9, digits=2)) GB: $(div(now * 100, total))% \r",
+        )
+    end
+end
+
+
+output_dir = basename(@__DIR__) * "_artifact"
+if isdir(output_dir)
+    @warn "$output_dir already exists. Content will end up in the artifact and may be overwritten."
+    @warn "Abort this calculation, unless you know what you are doing."
+else
+    mkdir(output_dir)
+end
+
+if !isfile(FILE_NAME)
+    @info "$FILE_NAME not found, downloading it (might take a while)"
+    # The server has poor certificates, so we have to disable verification
+    downloader = Downloads.Downloader()
+    downloader.easy_hook =
+        (easy, info) ->
+            Downloads.Curl.setopt(easy, Downloads.Curl.CURLOPT_SSL_VERIFYPEER, false)
+    println(FILE_URL)
+    println(FILE_NAME)
+    downloaded_file = Downloads.download(FILE_URL; downloader, progress = download_progress)
+    Base.mv(downloaded_file, FILE_NAME)
+end
+
+output_path = joinpath(output_dir, FILE_NAME)
+# delete the old output file if it exists
+if isfile(output_path)
+    Base.rm(output_path)
+end
+input_ds = NCDataset(FILE_NAME, "r")
+
+# calculate absolute temperature from climatology and anomaly
+climatology = input_ds["climatology"][:, :, :]
+temperature_anomaly = input_ds["temperature"][:, :, :]
+time = input_ds["time"][:]
+temperature_absolute = similar(temperature_anomaly)
+# loop through each time point, and calculate the absolute temperature at it
+for t = 1:input_ds.dim["time"]
+    month = round(Int, 12 * (time[t] % 1) + sqrt(eps(Float64)))
```

Comment:
```suggestion
    month = ceil(Int, 12 * (time[t] % 1) + sqrt(eps(Float64)))
```
I think `ceil` is safer here, and you may not need to sqrt(eps(FT)) anymore?